lib/run-tests/transpile.js

```diff
@@ -27,14 +27,16 @@ function getOptions(features, isModule, isStrict) {
     sourceType: isModule ? "module" : "script",
     highlightCode: false,
     generatorOpts: { minified: true },
-    parserOpts: { strictMode: isStrict }
   });
 
   configCaches.set(configKey, config);
   return config;
 }
 
 module.exports = function transpile(code, { features, isModule, isStrict } = {}) {
-  const ret = transformSync(code, getOptions(features || [], Boolean(isModule), Boolean(isStrict))).code;
+  if (isStrict && !isModule && !/^\s*['"]use strict['"]/.test(code)) {
+    code = `"use strict";\n${code}`;
```

Comment:
Maybe let's not add `\n` to avoid potentially confusing off-by-one line numbers in errors.

Reply:
We have `generatorOpts: { minified: true }` enabled, which probably doesn't matter. :)